modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1355,8 +1373,7 @@ void ONNXImporter::populateNet(Net dstNet)
                 CV_Assert(layer_id.find(node_proto.input(i)) == layer_id.end());
 
             String interp_mode = layerParams.get<String>("coordinate_transformation_mode");
-            CV_Assert_N(interp_mode != "tf_crop_and_resize", interp_mode != "asymmetric",
-                        interp_mode != "tf_half_pixel_for_nn");
+            CV_Assert_N(interp_mode != "tf_crop_and_resize", interp_mode != "tf_half_pixel_for_nn");
```

Comment:
I think that it works only for nearest but not bilinear upsampling.

Reply:
using mode `bilinear` creater the graph with attributes.
![Screenshot from 2020-04-12 19-46-21](https://user-images.githubusercontent.com/26292819/79070936-577f9700-7cf6-11ea-8370-aed39fd78941.png)

Reply:
Thank, ok, let's keep as is.